net-im/swiften/swiften-4.0.ebuild

```diff
@@ -0,0 +1,99 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit scons-utils toolchain-funcs
+
+DESCRIPTION="A C++ library for implementing XMPP applications"
+HOMEPAGE="https://www.swift.im/"
+SRC_URI="https://swift.im/downloads/releases/swift-${PV}/swift-${PV}.tar.gz"
+
+LICENSE="BSD BSD-1 CC-BY-3.0 GPL-3 OFL-1.1"
+SLOT="0/4"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug doc expat icu idn sqlite upnp zeroconf"
+
+DEPEND="dev-libs/boost:=
+	dev-libs/openssl:0=
+	sys-libs/zlib
+	doc? ( app-text/docbook-xsl-stylesheets
+		app-text/docbook-xml-dtd:4.5 )
+	expat? ( dev-libs/expat )
+	!expat? ( dev-libs/libxml2 )
+	icu? ( dev-libs/icu:= )
+	idn? ( net-dns/libidn )
+	sqlite? ( dev-db/sqlite:3 )
+	upnp? (	net-libs/libnatpmp
+		net-libs/miniupnpc )
+	zeroconf? ( net-dns/avahi )"
+
+RDEPEND="${DEPEND}"
+
+S="${WORKDIR}/swift-${PV}"
+
+src_prepare() {
+	default
+
+	# Remove those directories, as the system libs are used
+	rm -fr 3rdParty/Boost 3rdParty/DocBook 3rdParty/Expat 3rdParty/LibIDN 3rdParty/LibMiniUPnPc 3rdParty/LibNATPMP 3rdParty/Lua 3rdParty/OpenSSL 3rdParty/SCons 3rdParty/SQLite 3rdParty/Unbound 3rdParty/Zlib || die
+
+	# Remove 'Limber', 'Slimber' and 'Sluift', as compiling headless
+	rm -fr Limber Slimber Sluift || die
+}
+
+src_configure() {
+	MYSCONS=(
+		ar="$(tc-getAR)"
+		allow_warnings="yes"
+		assertions="$(usex debug yes no)"
+		build_examples="yes"
+		boost_bundled_enable="false"
+		boost_force_bundled="false"
+		cc="$(tc-getCC)"
+		ccache="no"
+		ccflags="${CFLAGS}"
+		coverage="$(usex debug yes no)"
+		cxx="$(tc-getCXX)"
+		cxxflags="${CXXFLAGS}"
+		debug="$(usex debug yes no)"
+		distcc="no"
+		$(usex doc "docbook_xml=${EPREFIX}/usr/share/sgml/docbook/xml-dtd-4.5" "" )
+		$(usex doc "docbook_xsl=${EPREFIX}/usr/share/sgml/docbook/xsl-stylesheets" "")
+		experimental="$(usex sqlite yes no)"
+		experimental_ft="$(usex upnp yes no)"
+		hunspell_enable="false"
+		install_git_hooks="no"
+		libidn_bundled_enable="false"
+		libminiupnpc_force_bundled="false"
+		libnatpmp_force_bundled="false"
+		link="$(tc-getCXX)"
+		linkflags="${LDFLAGS}"
+		max_jobs="no"
+		optimize="no"
+		qt5="false"
+		sqlite_force_bundled="false"
+		swiften_dll="false"
+		swift_mobile="no"
+		target="native"
+		test="none"
+		try_avahi="$(usex avahi true false)"
```

Comment:
it seems that you forgot to remove that?

Reply:
Whoops, thx. Shouldn't normally warn Repoman about this?